euleros/content.md

```diff
@@ -0,0 +1,35 @@
+%%LOGO%%
+
+# EulerOS
+
+EulerOS provides enhanced security, compatibility and easy-to-use features along with high reliability. It meets the increasing requirements of Linux OS in enterprise applications and provides a compelling choice of open IT platform for users. EulerOS integrates the advanced Linux technologies, delivers more value to enterprise users in terms of high capability, stability, usability and expansibility. And it helps you reallocate resources from maintaining the status quo to tackling new challenges.
+
+## Operating System
+
+Linux/Unix, Other V2.0
+
+## High Lights
+
+-	Extreme Security: EulerOS is one of the most secure operating systems available, which provides variety of security technologies to prevent intrusions and protect your system.
+-	Carrier-class Reliability, Availability and Serviceability: EulerOS customers are able to provide carrier-class ultra-long-term reliability and stability to their users.
+-	High Performance: EulerOS has done a lot of optimization in the compilation, virtual memory, CPU scheduling, IO drive, network and file system and so on.
+
+## License
+
+EulerOS is released under the GPLv2. See the `LICENSE` file in this repository for more information.
```

Comment:
"in this repository" won't make sense when this description is added to Docker Hub / Store -- perhaps this section should instead be removed in favor of the automatic section that'll be added with the `license.md` file?